docs/scripts/build_sxp_parquets.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+import sed
+from sed import SedProcessor
+from sed.dataset import dataset
+
+config_file = Path(sed.__file__).parent / "config/sxp_example_config.yaml"
+
+dataset.get("Au_Mica", root_dir="./tutorial")
+data_path = dataset.dir
+
+
+config_override = {
+    "core": {
+        "paths": {
+            "data_raw_dir": data_path,
+            "data_parquet_dir": data_path + "/processed/",
+        },
+    },
+}
+
+runs = [
```

Comment:
any specific reason for this sequenence?

Reply:
I think this was just copy-paste.